### DISCUSSION TITLE
Why all HTTP status codes are hardcoded right in the code?

### DISCUSSION THREAD

**Discussion by aikrom** [adonisjs/core#3535] [Ideas]
I saw that in class [Response ](https://github.com/adonisjs/http-server/blob/develop/src/Response/index.ts)all HTTP status codes are specified in methods. I want to suggest that all the status codes be separated into a some constants file.

I was writing tests now, and I need to write all the status codes right in the code, but I don’t remember all the HTTP status codes. It’s easier for to navigate by the name of the statuses.

For example, NestJs has a ["file"](https://github.com/nestjs/nest/blob/master/packages/common/enums/http-status.enum.ts) with all the codes and it is very useful.

P.S. It is not difficult to create a file with these constants in your project yourself. I just wanted to know why it's not done that way? Maybe there are reasons, is it better or something else?

**Comment by RomainLanz** [comment]
Happy to accept a PR 🤗

**Reply by aikrom** [reply]
I'll work on it
